internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/ReplicaStatus.java

```diff
@@ -120,8 +121,8 @@ private void addHistoricStatus(ExecutionStatus status) {
     removeOldStatuses();
 
     StatusSnapshot snapshot = new StatusSnapshot(status, LocalDateTime.now().toString());
-    if (!StringUtils.isEmpty(incrementalPushVersion)) {
-      snapshot.setIncrementalPushVersion(incrementalPushVersion);
+    if (isIncrementalPushStatus(status)) {
```

Comment:
This can't be right..
So the status can be an incremental push version?
I was thinking we should pass `incrementalPushVersion` as a separate param to function: `addHistoricStatus`.

Reply:
Yeah I was wrong......updated